tests/integration/test_load_model.py

```diff
@@ -0,0 +1,34 @@
+__copyright__ = "Copyright (c) 2021 Jina AI Limited. All rights reserved."
+__license__ = "Apache-2.0"
+
+from typing import Dict, Callable
+
+import pytest
+
+from jina import DocumentArray
+
+from jinahub.text.encoders.transform_encoder import TransformerTorchEncoder
+
+
+MODELS_TO_TEST = [
+    'sentence-transformers/distilbert-base-nli-stsb-mean-tokens',
+    'bert-base-uncased',
+    'distilroberta-base',
+    'distilbert-base-cased-distilled-squad',
+]
+
+
+@pytest.mark.parametrize(
+    'model_name', MODELS_TO_TEST
+)
+def test_load_torch_models(model_name: str, data_generator: Callable):
```

Comment:
this is not an integration test, this is a unit test

Reply:
https://github.com/jina-ai/executor-transformer-torch-encoder/pull/1/commits/9116b7c45458db6495de5eca7b8f2d31dd4104c1